contracts/src/CloneFactory.sol

```diff
@@ -22,30 +21,4 @@ contract CloneFactory {
             result := create(0, clone, 0x37)
         }
     }
-
-    function isClone(
```

Comment:
Deleted because this function is never used (this contract is manually copied from https://github.com/optionality/clone-factory/blob/master/contracts/CloneFactory.sol because of the solidity version conflicts).